HabboCity/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "author": {
+    "name": "Cold",
+    "id": "254638264076468234"
+  },
+  "service": "HabboCity",
+  "description": {
+    "fr": "HabboCity est un grand rétro Habbo français. Rejoins-nous et deviens célèbre, fais de nouvelles rencontres, tchates, construis, participe à des jeux et bien plus..."
+  },
+  "url": [
+    "www.habbocity.me",
+    "habbocity.me"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/IC0Md4k.png",
+  "thumbnail": "https://i.imgur.com/plMx986.png",
+  "color": "#FFEE51",
+  "tags": [
+    "other",
+    "random",
+    "habbo"
+  ],
+  "category": "other",
+  "iframe": false
```

Comment:
Remove this if you aren't using iframes.

Reply:
Done